ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -63,6 +63,36 @@ resources:
     requests:
       cpu: 100m
       memory: 200Mi
+  installer:
+    limits:
+      memory: 9Gi
+    requests:
+      cpu: "3"
+      memory: 5Gi
+  upi-installer:
```

Comment:
I don't understand how these keys tie us to the `openshift-install` compilation step.  Maybe we want to restore just the `bin` key I seem to have accidentally removed in 3f2f01c43b (#1677)?  Which was maybe originally from 0b1cdda50a (#891)?

Reply:
using it similar to unit https://github.com/openshift/release/pull/3588/files/c589ac71aed9634a04298e1b3126fab5b52a281a#diff-f9121dca038210d2d38e2713d851b4efR96

Reply:
> using it similar to unit

Yup, although I'd be surprised if wee needed 5 GiB memory for our unit tests.  And I'm still not clear how these are supposed to work, but I worry that this line is getting limits/requests for our UPI jobs (which are probably fine already) instead of for the compilation job that the UPI tests depend on.  Does the CI output show this worked?

Reply:
> Does the CI output show this worked?
yes the build jobs for rehearse had the correct resource requests/limits for build pods.